spec/classes/puppet_spec.rb

```diff
@@ -139,6 +139,25 @@
         end
       end#no params
 
+      context 'when allinone is true' do
+        let(:params) {{'allinone' => true}}
+        it 'should manage the facts directories' do
```

Comment:
If I move this inside the other test, then we'll have context "allinone is true" inside context "no params", which is also misleading. Pick your poison; I'm happy to do it either way.